packages/happner-cluster/lib/happner-cluster.js

```diff
@@ -57,7 +57,7 @@ module.exports.create = require('util').promisify(function (config, callback) {
   cursor = config.happn.services.subscription;
   cursor.config = cursor.config || {};
   cursor.config.filter = filterEventVersions;
-
+  if ((config.port || config.port) === 0 && !config.happn.port) config.happn.port = config.port;
```

Comment:
we may be able to simplify this a bit, ie: if (!config.happn.port) { config.happn.port =  config.port }

Reply:
done